src/main/java/com/liferay/faces/util/render/internal/FacesURLEncoderMyFacesImpl.java

```diff
@@ -34,11 +34,11 @@
 	// Logger
 	private static final Logger logger = LoggerFactory.getLogger(FacesURLEncoderMyFacesImpl.class);
 
-	// Private Constants
-	private final Method MYFACES_METHOD_ENCODE_URI_ATTRIBUTE;
+	// Private Final Data Members
+	private final Method myfacesMethodEncodeURIAttribute;
```

Comment:
@stiemannkj1 As I recall, usage of the final keyword for private data members is not commonly used in the codebase.